tonic/models/vic/grid_params.py

```diff
@@ -1357,6 +1426,11 @@ def veg_class(vegl_file, c=Cols(veglib_fcan=False)):
                 for col in np.arange(0, col_desc):
                     data[row - 1][:] = words[:col_desc]
                     data[row - 1].append(sep.join(words[col_desc:]))
+                if veglib_photo:
+                    if data[row - 1][c.veglib['lib_Ctype']] == 'C3':
+                        data[row - 1][c.veglib['lib_Ctype']] = 0
+                    elif data[row - 1][c.veglib['lib_Ctype']] == 'C4':
+                        data[row - 1][c.veglib['lib_Ctype']] = 1
```

Comment:
I don't get this. Let's think about how to continue using the string representation.  It is certainly easier to ready :wink: